components/dashboard/src/settings/Integrations.tsx

```diff
@@ -302,7 +301,7 @@ function GitProviders() {
                         <span className="text-sm my-auto text-gray-400">Permissions</span>
                     </div>
                     <div className="my-auto flex w-1/12 mr-4 opacity-0 group-hover:opacity-100 justify-end">
-                        <div className="self-center hover:bg-gray-200 rounded-md cursor-pointer w-8">
+                        <div className="self-center hover:bg-gray-200 dark:hover:bg-gray-700 rounded-md cursor-pointer w-8">
                             <ContextMenu menuEntries={gitProviderMenu(ap)}>
                                 <img className="w-8 h-8 p-1" src={ThreeDots} alt="Actions" />
```

Comment:
**issue**: This also needs to use a ligher color like the `X` button on the modal.

Reply:
@gtsiolis Going with `text-gray-600` in light mode (as currently) and `text-gray-100` in dark mode. Let me know if this works. 😁

Reply:
THE. BEST. GRAY.